### PR TITLE
cleanup(ci): make use of falcosecurity/testing provided composite action

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind cleanup

**Any specific area of the project related to this PR?**

/area CI

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by FedeDP** [comment]
/milestone 0.38.0

also cc @jasondellaluce  i am not sure whether we still need the `testing` submodule at this stage.

File: .github/workflows/reusable_test_packages.yaml [hunk 1]
```diff
@@ -43,54 +32,20 @@ jobs:
...
+      - name: Run tests
+        uses: falcosecurity/testing@main    
         with:
-          paths: "submodules/falcosecurity-testing/report.xml"
-          show: "fail"
+          test-falco: 'true'
+          test-falcoctl: 'true'
+          test-k8saudit: 'true'
+          test-static: ${{ inputs.static && 'true' || 'false' }}
+          test-drivers: ${{ inputs.arch == 'x86_64' && 'true' || 'false' }}
```
Comment (FedeDP) [inline .github/workflows/reusable_test_packages.yaml:50]: Keep drivers test on aarch64 disabled since `actuated` nodes gave some issues in the past.

**Review by jasondellaluce — APPROVED** [review]
/approve

**Comment by FedeDP** [comment]
Will cleanup it in a followup PR ;)

**Review by leogr — APPROVED** [review]

**Pull request merged**